src/components/html/EcOrderStep.html

```diff
@@ -0,0 +1,35 @@
+<div class="ec-order-steps">
+    <ul>
+      <li class="complete">
+        <div class="order__step">{{i18n('OrderCreated')}}
```

Comment:
tem que manter a classe original `ec-order-steps`:

```suggestion
        <div class="ec-order-steps__step">{{i18n('OrderCreated')}}
```

Reply:
Ok